bundles/redhat8.6/packages.txt.gotmpl

```diff
@@ -15,16 +15,18 @@ yum-utils
 cloud-init
 cloud-utils-growpart
 container-selinux
+{{ if .FetchKubernetesRPMs -}}
 kubectl-{{ .KubernetesVersion }}-0
 kubelet-{{ .KubernetesVersion }}-0
 kubeadm-{{ .KubernetesVersion }}-0
+cri-tools
+{{- end }}
```

Comment:
The `util-linux` [dependency](https://github.com/mesosphere/dkp-release/blob/30ee3913f9b8c8fee69dda891de748babf913d83/k8s-build/rpms/kubelet.spec#L35C10-L35C21) is not explicitly defined here in the list but it is [being pulled correctly in the e2e test](https://github.com/mesosphere/konvoy-image-builder/actions/runs/10368285418/job/28720836108?pr=1140#step:9:1754). All other deps are here in the list.